Symfony/CS/Fixer/Symfony/NoTrailingWhitespaceInCommentFixer.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Symfony;
+
+use Symfony\CS\AbstractAnnotationRemovalFixer;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ */
+final class NoTrailingWhitespaceInCommentFixer extends AbstractAnnotationRemovalFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        foreach ($tokens as $token) {
+            if ($token->isComment()) {
+                $token->setContent(
+                    preg_replace('/[ \t]+$/m', '', $token->getContent())
+                );
+            }
+        }
+
+        return $tokens->generateCode();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getDescription()
+    {
+        return 'There MUST be no trailing spaces inside comments.';
```

Comment:
A the set of doc comment can't possibly be a subset of the set of comments because surely that would mean that they'd both have to be tokenized as comments?

Anyway, I think we both understand each other's points, lol.